src/Common/src/Interop/Windows/mincore/Interop.GetFullPathNameW.cs

```diff
@@ -9,10 +10,30 @@ partial class Interop
 {
     partial class mincore
     {
+        /// <summary>
+        /// WARNING: This overload does not implicitly handle long paths.
+        /// </summary>
```

Comment:
Do you have any suggestions for naming? GetFullPathNameNoLongPath? This one in particular is used for the stack allocating code in PathHelper (all under MAX_PATH).

Like I mentioned, I intend to pass through these P/Invokes and normalize the naming. I want to drop the "W", use "Private" (anything better for that?), add comments on the privates to avoid leaking in other P/Invoke wrappers.